prolif/plotting/network/network.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <script type="text/javascript" src="https://unpkg.com/vis-network@9.0.4/dist/vis-network.min.js"></script>
+    <link href="https://unpkg.com/vis-network@9.0.4/dist/dist/vis-network.min.css" rel="stylesheet" type="text/css" />
+    <link href="network.css" rel="stylesheet" type="text/css" />
```

Comment:
unfortunately referencing the file won't work, I suggest you use inline `<style>` instead and fill it with a new variable `%(css)s` or something like that